tools/harness-thci/README.md

```diff
@@ -6,51 +6,51 @@ which is used by Thread Test Harness Software to control OpenThread-based refere
 scenario.
 
 Currently, there are two THCI implementations for OpenThread. One is for OpenThread CLI which is based on the CC2538 
-example platform and is included in the current Thread Test Harness Software release. The other is for OpenThread WpanCtl
-which is based on wpantund running on a linux host (For example: Raspberry Pi 3B) working with a NCP (Network Co-Processor) 
-board (For example: Nordic Semiconductor nRF52840) running a OpenThread NCP image.
+example platform and is included in the current Thread Test Harness Software release. The other is for OpenThread wpanctl
+which is based on wpantund running on a linux host (for example: Raspberry Pi 3B) working with a NCP (Network Co-Processor) 
+board (for example: Nordic Semiconductor nRF52840) running a OpenThread NCP image.
 
 Platform developers should modify the THCI implementation directly to match their platform (e.g. serial baud rate).
```

Comment:
Change the e.g. to "for example"

(for example, serial baud rate)

Reply:
@FiveDimensions Don't forget to update this, please.

Reply:
updated, thanks for reminding